src/Integration.Vsix.UnitTests/ErrorList/SonarErrorListDataSourceTests.cs

```diff
@@ -44,6 +47,38 @@ public void TestInitialize()
             mockTableManagerProvider.Setup(x => x.GetTableManager(StandardTables.ErrorsTable)).Returns(mockTableManager.Object);
         }
 
+        [TestMethod]
+        public void MefCtor_CheckExports()
+        {
+            CompositionBatch batch = new CompositionBatch();
```

Comment:
Cool, TIL

Reply:
Me too - I had to look up what "TIL" means ;-)